nextest-runner/src/target_runner.rs

```diff
@@ -148,6 +161,10 @@ impl TargetRunner {
                 configs.push(config);
             }
 
+            if dir == workspace_root {
+                break;
+            }
```

Comment:
Hmm, I don't think this is quite correct -- by default, cargo does search past the workspace root: https://doc.rust-lang.org/cargo/reference/config.html. So I think there should be two modes:
1. default mode, which emulates cargo
2. isolation mode, which stops at the workspace root and doesn't look at the home directory.